src/core/Object3D.js

```diff
@@ -25,6 +25,110 @@ const _zAxis = /*@__PURE__*/ new Vector3( 0, 0, 1 );
 const _addedEvent = { type: 'added' };
 const _removedEvent = { type: 'removed' };
 
+var skipUMWToAvoidLoop = false;
```

Comment:
Please make this variable just `let`.

Reply:
OK.  Also updating variable name to _skipUMWToAvoidLoop, which I believe matches the convention for module-scoped private variables.

Reply:
When the deprecated `this.updateMatrixWorld !== Object3D.prototype.updateMatrixWorld` clause will be removed (11 releases from the one this PR will be merged in) this variable could be removed, correct?

Reply:
Agreed.